### DISCUSSION TITLE
Multiple Query Result

### DISCUSSION THREAD

**Discussion by rifqiabidin** [tconbeer/harlequin#617] [Ideas (Feature Requests)]
Since we got multiple query editor, I think it's great if we can get multiple query result window like dbeaver did?
![image](https://github.com/user-attachments/assets/9d87a25c-e21f-4a99-907c-6dd345c2dbf9)

**Comment by tconbeer** [comment]
We support multiple query results - you just have to select (highlight) multiple queries in the editor before executing them. Do you mean you want to keep old results around instead of deleting those tabs?

**Reply by rifqiabidin** [reply]
Yes, exactly. Sorry I forgot to highlight the attached screenshot on previous post.
